packages/asset-swapper/src/utils/market_operation_utils/orders.ts

```diff
@@ -318,6 +321,11 @@ export function createBridgeDataForBridgeOrder(order: OptimizedMarketBridgeOrder
             const lidoFillData = (order as OptimizedMarketBridgeOrder<LidoFillData>).fillData;
             bridgeData = encoder.encode([lidoFillData.stEthTokenAddress]);
             break;
+        case ERC20BridgeSource.Firebird:
+            const firebirdFillData = (order as OptimizedMarketBridgeOrder<FirebirdFillData>).fillData;
+            bridgeData = encoder.encode([firebirdFillData.router, firebirdFillData.pool]);
```

Comment:
From my previous comment this would be
```suggestion
            bridgeData = encoder.encode([firebirdFillData.router, [firebirdFillData.pool]]);
```

Reply:
handled in 25434c0